lib/docker/image.rb

```diff
@@ -113,8 +113,12 @@ def create(opts = {}, creds = nil, conn = Docker.connection, &block)
         :response_block => response_block(body, &block)
       )
       json = Docker::Util.fix_json(body)
-      image = json.reverse_each.find { |el| el && el.key?('id') }
-      new(conn, 'id' => image && image.fetch('id'), :headers => headers)
+      completions = json.compact.select { |j| j['status'] && j['status'].include?('complete') }
```

Comment:
I think I prefer this since there is a LOT of messages that get returned and usually only a few matter here.